docs/docs/api/Tasks/fetchTaskByDay.md

```diff
@@ -7,69 +7,56 @@ borderRadius: '2px',
 color: '#E83E8C',
 }}>{children}</span> );
 
-<Endpoint>GET tasks/day/:day/</Endpoint>: Fetching a single Task
+<Endpoint>GET /tasks/day/:day</Endpoint>: Fetching a single Task
 
 ```json
-{
-
-}
+{}
 ```
 
 ### Example Request
+
 This is a **protected route**, a **valid JWT is required** in the header field
+
 #### Header
+
 ```
 Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
 
 #### Body
-```json
-{
 
-}
+```json
+{}
 ```
 
 #### Parameters
```

Comment:
Just like in the `fetchSingleTask`, we can remove the JSON since that's not part of the parameters (and it's a GET request). It should be in a similar format to the way it was done in `fetchSingleTask`.

If I'm not mistaken, the day shouldn't be in ISO format no? I believe we agreed that it should be something like `/tasks/day/20220226` (year month day)

Reply:
Please double check to make sure that the documentation is accurately portraying the API